src/api/spec/bootstrap/features/webui/packages_spec.rb

```diff
@@ -12,6 +12,11 @@
     let(:project_path) { package_show_path(project: user_tab_user.home_project, package: package) }
   end
 
+  before do
```

Comment:
If you have this `before` block for the login, then in the `describre` is not needed to [re-login](https://github.com/openSUSE/open-build-service/pull/7601/files#diff-abb1fd0f116ff104981dcf69d4f0bf88R30) again, right?

Reply:
Good point:thumbsup: